doc/source/user_guide/timeseries.rst

```diff
@@ -1628,24 +1600,29 @@ labels.
 
    ts.resample('5Min', label='left', loffset='1s').mean()
 
-.. note::
+.. warning::
 
-    The default values for ``label`` and ``closed`` is 'left' for all
+    The default values for ``label`` and ``closed`` is '**left**' for all
     frequency offsets except for 'M', 'A', 'Q', 'BM', 'BA', 'BQ', and 'W'
     which all have a default of 'right'.
 
+    This might lead to unintended look-ahead bias as in the following example:
```

Comment:
Is "look-ahead bias" a common term?

Reply:
It is in the context of timeseries forecasting

Reply:
Dropping bias, and perhaps adding a small definition. Something like

    This might lead to unintended looking ahead, where the value for a later date (Sunday) is pulled back to
    a previous date (Friday).